src/main/resources/db/migration/V12__add_failure_code_failed_devices.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE device_updates ADD COLUMN failure_code VARCHAR(255) NULL;
```

Comment:
Why is it safe to assume that failure code won't exceed 255 chars? Do we have a strict limitation on it somewhere?

Reply:
If there is any limit specified I'm not aware of it. It just seemed to me 255 should be enough given the (few) uses I've seen. We could bump it up one or two orders of magnitude I guess? Or what would be a reasonable limit/type here? @jerrytrieu @koshelev